com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -952,6 +960,30 @@ private void downloadSymbols(IProgress progress) throws IOException, CompileExce
 
                     BundleHelper.throwIfCanceled(monitor);
 
+                    IProgress m = monitor.subProgress(99);
+
+                    m.beginTask("Reading classes...", 1);
+                    Bob.createClassLoaderScanner();
+                    registerPipelinePlugins();
+                    this.scan(Bob.getClassLoaderScanner(), "com.dynamo.bob");
+                    this.scan(Bob.getClassLoaderScanner(), "com.dynamo.bob.pipeline");
+                    m.done();
```

Comment:
Now, when the plugins are built, we can register the classes from those .jar files.